lib/granite/action/subject.rb

```diff
@@ -29,15 +29,19 @@ def subject(name, *args, &block)
 
           self._subject = name
         end
+
+        def subject?
+          _subject.present?
+        end
       end
 
       def initialize(*args)
-        if self.class._subject.blank?
+        unless self.class.subject?
           super
           return
         end
```

Comment:
Not strictly necessary since it's not your code, but I think it would be better to change this to
```suggestion
        return super unless self.class.subject?
```

Reply:
:+1:

Reply:
Need to disable rubocop:
```ruby
return super unless self.class.subject? # rubocop:disable Lint/ReturnInVoidContext
```